bazel/defs.bzl

```diff
@@ -185,6 +185,7 @@ if [ ! -z ${{DEBUG+x}} ]; then
 fi
 
 avrdude -v -P usb -p {part} -F \\
+        -C /home/manu/Desktop/Formula/olin-electric-motorsports/.avrduderc \\
```

Comment:
I spent a couple hours in bazel docs and cant quite figure out how to refer to the repo root in bazel/skylark world. Would appreciate any help

On a different note, using this path does make avrdude use this file successfuly